edk2toolext/capsule/pyopenssl_signer.py

```diff
@@ -21,10 +22,30 @@ def sign(data, signature_options, signer_options):
     dictionaries that are used by capsule_tool and capsule_helper
```

Comment:
Might as well add the type annotations/hints. ^^^

Reply:
Added, tested all configurations, no breaks (of my code anyway).

Reply:
Passed unit tests as well, resolved in commit d16b0339e4caaab3d6b7f0bd18027f8b52b34cf2